satellite/satellitedb/export_test.go

```diff
@@ -0,0 +1,7 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package satellitedb
```

Comment:
won't this file export `Overlaycache` and `LockedOverlaycache` for anything that imports `satellitedb` now? I would assume we only want to access these items inside tests and don't want to expose them for other use cases.

Reply:
It doesn't actually look like you are using these anywhere. Do we need this file?

Reply:
`Overlaycache` and `LockedOverlaycache` would be exported but `export_test` shouldn't be included in any non-test build. You're right though, we didn't need this in the end.